waspc/examples/todoApp/src/client/pages/ProfilePage.tsx

```diff
@@ -13,10 +14,38 @@ export const ProfilePage = ({
 }: {
   user: User
 }) => {
+  const [messages, setMessages] = useState<
+    { id: string; username: string; text: string }[]
+  >([])
+  const { socket, isConnected } = useSocket()
```

Comment:
I'm very happy that this supports full stack type safety, that was my first suggestion after seeing the example in the PR's description. Nice work to everyone involved!